main.py

```diff
@@ -0,0 +1,58 @@
+if __name__ == "__main__":
+    from bs4 import BeautifulSoup
+    from fpdf import FPDF
+
+    pdf = FPDF('P', 'mm', 'Letter')
+    pdf.add_page()
+    pdf.set_font('helvetica', 'B', 20)  
+
+    pdf.cell(0, 10, 'Liste av sist oppdatert CVer', ln=True) 
+
+    pdf.set_font('helvetica', '', 16)  
+
+    req = "C:\\Users\\sldobler\\Downloads\\CV-partner.html"
+    with open(req, 'r', encoding='utf-8') as htmlFile:
```

Comment:
Implementing error handling with output messaging to enhance user-friendliness and robustness. check if the file exist